src/pages/data-visualization/getting-started/react.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: Get started
+tabs: ['Vanilla', 'React', 'Angular', 'Vue', 'Other frameworks']
+---
+
+import DataOptionsSection from './shared-data-options-section';
+
+<AnchorLinks>
+
+<AnchorLink>Carbon Charts React library</AnchorLink>
+<AnchorLink>Install</AnchorLink>
+<AnchorLink>Getting started</AnchorLink>
+<AnchorLink>Development</AnchorLink>
+<AnchorLink>Troubleshooting</AnchorLink>
+
+</AnchorLinks>
+
+## Resources
+
+<Row className="resource-card-group">
+
+<Column colMd={4} colLg={4} noGutterSm>
+  <ResourceCard
+    subTitle="Try Carbon Charts React components with CodeSandbox."
+    href="https://codesandbox.io/s/pppmo3ollx">
+    <MdxIcon name="codesandbox" />
+  </ResourceCard>
+</Column>
+
+  <Column colMd={4} colLg={4} noGutterSm>
+    <ResourceCard
+      subTitle="Carbon Charts React"
+      href="https://github.com/carbon-design-system/carbon-charts/tree/master/packages/react"
+    >
+      <MdxIcon name="github" />
+    </ResourceCard>
+  </Column>
+</Row>
+
+## Carbon Charts React library
+
+The library provides front-end developers & engineers a collection of reusable charting components to build websites and user interfaces. Adopting the library enables developers to use consistent markup, styles, and behavior in prototype and production work.
+
+## Install
+
+#### Using yarn:
+
+<br />
+
+```bash
+$ yarn add @carbon/charts @carbon/charts-react d3
+```
+
+#### If you prefer npm:
+
+<br />
+
+```bash
+$ npm install --save @carbon/charts @carbon/charts-react d3
+```
+
+## Getting started
+
+To start using the `BarChart` component, try the example below:
+
+```jsx
+import React from 'react';
+
+import { defaultColors } from '@carbon/charts';
+import { BarChart } from '@carbon/charts-react';
+import '@carbon/charts/style.css';
+
+const stackedBarOptions = {
+  // ...see next section
+};
+
+class App extends React.Component {
+  state = {
+    stackedBarData: {
+      // ...see next section
+    },
+  };
+
+  render() {
+    return (
+      <div className="App">
+        <BarChart
+          data={this.state.stackedBarData}
+          options={stackedBarOptions}
+          width="100%"
+          height={400}
+        />
+      </div>
+    );
+  }
+}
+
+// ...
+```
+
+<DataOptionsSection />
+
+## Development
+
+Please refer to the [Contribution Guidelines](https://github.com/carbon-design-system/carbon-charts/blob/master/CONTRIBUTING.md) before starting any work.
+
+### Using the server
+
+We recommend the use of [React Storybook](https://github.com/storybookjs/storybook/tree/next/app/react) for developing components.
+
+#### Start the server:
```

Comment:
Is it just `h4`'s that are getting a semicolon? Not sure if this was intentional or not. The other headers don't have one.